documentation/library_guide/overview.rst

```diff
@@ -1,14 +1,14 @@
-oneDPL overview
-###############
+|onedpl_long| Overview
+#######################################
 
-Parallel API can be used with `C++ Standard Execution
+Parallel API can be used with the `C++ Standard Execution
 Policies <https://en.cppreference.com/w/cpp/algorithm/execution_policy_tag_t>`_
-to enable parallelism on the host.
+to enable parallelism on your host.
```

Comment:
```suggestion
to enable parallelism on the host.
```
I suggest to keep "the host",  as it seems like a stable term in SYCL documentation.